ctapipe/io/simteleventsource.py

```diff
@@ -64,15 +74,22 @@ def __init__(self, config=None, parent=None, **kwargs):
         self.file_ = SimTelFile(
             self.input_url,
             allowed_telescopes=self.allowed_tels if self.allowed_tels else None,
-            skip_calibration=self.skip_calibration_events
+            skip_calibration=self.skip_calibration_events,
+            zcat=not self.back_seekable,
         )
+        if self.back_seekable and self.is_stream:
+            raise IOError('back seekable was required but not possible for inputfile')
 
         self._subarray_info = self.prepare_subarray_info(
             self.file_.telescope_descriptions,
             self.file_.header
         )
         self.start_pos = self.file_.tell()
 
+    @property
+    def is_stream(self):
```

Comment:
should this be a general property of `EventSource` rather than `SimTelEventSource`? (same comment also for the `back_seekable` trait)

Reply:
It is. This is inherited from EventSource but overriden